.github/workflows/terraform-plan.yml

```diff
@@ -91,7 +91,7 @@ jobs:
       fmt_outcome: ${{ steps.fmt.outcome }}
       init_outcome: ${{ steps.init.outcome }}
       validate_outcome: ${{ steps.validate.outcome }}
-      validate_output: ${{ steps.validate.outputs.stdout }}
+      validate_output: ${{ steps.validate.outputs.stdout || steps.validate.outputs.stderr }}
```

Comment:
This looks extremely reasonable to me, and feel free to ignore this comment and merge. 

Only question is whether there's an easy way to ensure we get both stdout and stderr if both are present, or at least to preference stderr so we always make sure we're seeing errors. So either (a) flipping the order of the two arguments around the `||` operator, or (b) possibly using the [`join` expression GHA supports](https://docs.github.com/en/actions/learn-github-actions/expressions#join), though I have no experience with that.

Reply:
Since there's no documented guarantee that only one of stderr or stdout will receive output (and at the very least, there are cases where `terraform plan` produces both), I updated this to always include output from both streams. You can see what this looks like in the updated [OP](https://github.com/usdigitalresponse/usdr-gost/pull/3081#issue-2311710957).